src/components/CuratorDashboard/CuratorProfile/CuratorProfileCard.tsx

```
@@ -0,0 +1,318 @@
+						(onChainIdentity as any)[key] = value;
+						setAddressWithIdentity(getEncodedAddress(address, network) || '');
+					} else if (!(onChainIdentity as any)[key]) {
+						(onChainIdentity as any)[key] = value;
+					}
+				}
+			});
+			setOnChainIdentity(onChainIdentity);
+		};
+		fetchIdentityInformation();
+		return () => {
+			unsubscribes && unsubscribes.length > 0 && unsubscribes.forEach((unsub) => unsub && unsub());
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)